demo/src/App.js

```diff
@@ -164,6 +172,18 @@ function App() {
           )}
           initialViewState={initialViewState}
           colormap={colormapOn}
+          overview={
+            overviewOn
+              ? new VivViewerOverview({
+                  viewWidth,
+                  viewHeight,
+                  loader,
+                  offset: OFFSET,
+                  overviewScale: 0.7
+                })
+              : null
+          }
+          overviewScale={0.7}
```

Comment:
I think this is a bit awkward. Identical props are passed to both `VivViewer` and `VivViewerOverview` (`viewWidth`, `viewHeight`, `loader`, and `overviewScale`(?)). I'd prefer we wrap this within `VivViewer`.

using defaults:
```javascript
        <VivViewer
          loader={loader}
          minZoom={MIN_ZOOM}
          viewHeight={viewHeight}
          viewWidth={viewWidth}
          sliderValues={sliderValues}
          colorValues={colorValues}
          channelIsOn={channelIsOn}
          initialViewState={initialViewState}
          colormap={colormapOn}
          overview // use defaults
        />
```

overriding specific properties:

```javascript
        <VivViewer
          loader={loader}
          minZoom={MIN_ZOOM}
          viewHeight={viewHeight}
          viewWidth={viewWidth}
          sliderValues={sliderValues}
          colorValues={colorValues}
          channelIsOn={channelIsOn}
          initialViewState={initialViewState}
          colormap={colormapOn}
          overview={{
              offset: OFFSET,
              scale: 0.7
          }} 
        />
```

Reply:
That makes sense.  I misinterpreted "go object oriented" mandate with respect to this